lib/themes/dosomething/paraneue_dosomething/scss/patterns/_button.scss

```diff
@@ -1,14 +1,73 @@
+%animated {
```

Comment:
Actually, I can just use the Forge one and then just override where necessary. Good call @DFurnes 💃